### ISSUE TITLE
Return status if cell number is invalid

### ISSUE THREAD

**Issue by obinakiller** [Ankit404butfound/PyWhatKit#42]
Is there a way to return a status when the message was not sent? For example, the mobile number does not exist or is invalid or not a valid whatsapp number.

Thanks

![Sem título](https://user-images.githubusercontent.com/46095048/107953349-ca424a00-6f79-11eb-9f56-df014bb38e2d.jpg)
.

**Comment by aaryanrr** [comment]
I dont think so that this is possible because there is no way you can determine wether a message has been sent or not. It would have been possible if there was an API. But i am not sure about this.

**Comment by Ankit404butfound** [comment]
I can create a flask API for that, only problem is that I don't have any spare numbers. Let's see what can be done.

**Comment by aaryanrr** [comment]
WhatsApp web shows this if the specified number is not found.

![Screenshot (88) - Copy](https://user-images.githubusercontent.com/73213670/108028570-0fd24600-7052-11eb-9551-c56c742ce09f.png)

So there must be a way to determine that this box popped up or not.

**Comment by Ankit404butfound** [comment]
> WhatsApp web shows this if the specified number is not found.
> 
> ![Screenshot (88) - Copy](https://user-images.githubusercontent.com/73213670/108028570-0fd24600-7052-11eb-9551-c56c742ce09f.png)
> 
> So there must be a way to determine that this box popped up or not.

Yes, we can simply call the source of the page page when there is this pop up and we can simply check if the source code contains "Phone number shared by url...". Only way of doing this is to use selenium and whatsapp doesn't allow this either so I need to keep it as a separate project so in case someone report it, I won't have to remove pywhatkit.

**Comment by aaryanrr** [comment]
Oh I see. Let's see if there is any other way because i can't find anything related to this.

**Comment by obinakiller** [comment]
I believe that this functionality would be very beneficial, as it would avoid not sending it due to human error, for example.

**Comment by aaryanrr** [comment]
@obinakiller Yes it would be beneficial. We are looking for a safe way to implement this.

**Comment by Naman23-coder** [comment]
maybe we can take a screenshot at the time for sending message and use opencv for text recognition

**Comment by Naman23-coder** [comment]
https://nanonets.com/blog/ocr-with-tesseract/ but it would take extra libraries

**Comment by aaryanrr** [comment]
I don't think so using OCR would be an efficient way to do this.

**Comment by pellife** [comment]
so, any updates about this? i'm having the same issue using pywhatkit and couldn't find any way to resolve it
